fml/platform/fuchsia/message_loop_fuchsia.h

```diff
@@ -25,6 +27,10 @@ class MessageLoopFuchsia : public MessageLoopImpl {
   void WakeUp(fml::TimePoint time_point) override;
 
   zx_handle_t timer_;
```

Comment:
Let's prefer `zx::timer` from `zx/timer.h` which is an RAII wrapper around these handles.